.github/workflows/build-and-test.yml

```diff
@@ -77,57 +73,36 @@ jobs:
                   path: data/invest-test-data
                   key: git-lfs-testdata-${{ hashfiles('Makefile') }}
 
-            - name: Set up Windows python
-              if: matrix.os == 'windows-latest'
-              uses: actions/setup-python@v1
-              with:
-                python-version: ${{ matrix.python-version }}
-                architecture: ${{ matrix.python-architecture }}
-
-            - name: Install build dependencies Windows
-              shell: bash -l {0}
-              if: matrix.os == 'windows-latest'
-              run: |
-                  python -m pip install --upgrade pip pytest setuptools toml twine ${{ matrix.numpy }}
-                  pip install $(python -c "import toml;print(' '.join(toml.load('pyproject.toml')['build-system']['requires']))")
-
-            - name: Set up MacOS python
-              if: matrix.os != 'windows-latest'
+            - name: Setup conda environment
               uses: goanpeca/setup-miniconda@v1.1.2
               with:
-                  activate-environment: mac-env
+                  activate-environment: ${{ matrix.os }}-test-env
                   auto-update-conda: true
                   python-version: ${{ matrix.python-version }}
                   channels: conda-forge
 
-            - name: Install build dependencies MacOS
-              if: matrix.os != 'windows-latest'
+            - name: Install build dependencies
               shell: bash -l {0}
               run: |
+                  conda install nomkl # make sure numpy is w/out MKL
                   conda upgrade -y pip setuptools
-                  pip install toml twine ${{ matrix.numpy }}
-                  pip install $(python -c "import toml;print(' '.join(toml.load('pyproject.toml')['build-system']['requires']))")
+                  conda install ${{ matrix.numpy }}
+                  python -m pip install scipy
```

Comment:
It may not matter much when it's installed, but is scipy really a build dependency?

Reply:
Good catch, fixed here: https://github.com/natcap/invest/pull/314/commits/b66ed3ba05fa330441d6ecc50135d1da89ab4a51